app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
```

Comment:
This class is missing some methods in regards to authentication. Specially the `forget` method that should be used to `update` the user's `remember_token` to `nil`.

[**REQUIRED**]